jax/experimental/jax2tf/examples/tf_js/quickdraw/README.md

```diff
@@ -0,0 +1,48 @@
+# Quickdraw
+
+This directory contains the code necessary to train a CNN on the Quickdraw
+dataset using Flax, and subsequently convert it using jax2tf into a model that
+can be loaded in TF.js.
+
+## Training the model
+
+You can train and export the model yourself by running
+```bash
+$ python3 quickdraw.py
+```
+
+The training itself lasts roughly 30 minutes, assuming the dataset has already
+been downloaded. You can also skip to the [next section](https://github.com/google/jax/blob/master/jax/experimental/jax2tf/examples/tf_js/quickdraw/README.md#interacting-with-the-model)
```

Comment:
nit: In this particular case you can just write [#interacting-with-the-model] because that works both in github and in Google.